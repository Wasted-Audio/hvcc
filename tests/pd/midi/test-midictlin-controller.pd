#N canvas 864 247 503 302 12;
#X obj 98 132 print;
#X obj 98 61 ctlin 13;
#X obj 98 96 pack f f;
#X connect 1 0 2 0;
#X connect 1 1 2 1;
#X connect 2 0 0 0;
