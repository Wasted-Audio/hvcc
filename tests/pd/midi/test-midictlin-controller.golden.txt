[@ 0.000] print: 59 3
[@ 0.000] print: 73 1
[@ 0.000] print: 60 3
[@ 0.000] print: 74 1
[@ 0.000] print: 61 3
[@ 0.000] print: 75 1
[@ 0.000] print: 62 3
[@ 0.000] print: 76 1
[@ 0.000] print: 63 3
[@ 0.000] print: 77 1
[@ 0.000] print: 62 3
[@ 0.000] print: 78 1
[@ 0.000] print: 63 3
[@ 0.000] print: 79 1
[@ 0.000] print: 62 3
[@ 0.000] print: 77 1
[@ 0.000] print: 61 3
[@ 0.000] print: 76 1
[@ 0.000] print: 60 3
[@ 0.000] print: 75 1
[@ 0.000] print: 59 3
[@ 0.000] print: 76 1
[@ 0.000] print: 58 3
[@ 0.000] print: 75 1
[@ 0.000] print: 74 1
[@ 0.000] print: 73 1